`azurerm_spring_cloud_configuration_service` - support for the `ca_certificate_id` property (#22814)
hashicorp/terraform-provider-azurerm: website/docs/r/spring_cloud_configuration_service.html.markdown

* `uri` - (Required) Specifies the URI of the repository.

> * `ca_certificate_id` - (Optional) Specifies the ID of the CA Spring Cloud Certificate for https URL of Git repository.

Reviewer: Presumably, this is:

```suggestion
* `ca_certificate_id` - (Optional) Specifies the ID of the Certificate Authority used when retrieving the Git Repository via HTTPS.
```
